collector/mdadm_linux.go

```diff
@@ -138,12 +163,20 @@ func parseMdstat(mdStatusFilePath string) ([]mdStatus, error) {
 		}
 		currentMD = mainLine[0]               // name of md-device
 		isActive := (mainLine[2] == "active") // activity status of said md-device
+		personality = mainLine[3]             // Personality type of md-device.
 
 		if len(lines) <= i+3 {
```

Comment:
Is this still a valid assumption with raid0? If a raid0 configuration appears at the end of a file, it seems there will only be 2 lines following?